tests/test_find_builds_cli.py

```diff
@@ -44,12 +43,45 @@ def test_attached_errata_succeed(self):
         fake_errata = flexmock(model=elliottlib.errata, get_metadata_comments_json=lambda: 1234)
         fake_errata.should_receive("get_metadata_comments_json").and_return(metadata_json_list)
 
-        builds = flexmock(Build(nvr="test-1.1.1"))
+        builds = flexmock(Build(nvr="test-1.1.1", product_version="RHEL-7-OSE-4.5"))
         builds.should_receive("attached_to_open_erratum").and_return(True)
         builds.should_receive("open_errata_id").and_return([12345])
 
         # expect return list with one build
-        self.assertEqual([Build("test-1.1.1")], _attached_to_open_erratum_with_correct_product_version([builds], "RHEL-7-OSE-4.5", fake_errata))
+        self.assertEqual([Build("test-1.1.1")], _attached_to_open_erratum_with_correct_pv("image", [builds], fake_errata))
+
+    def test_get_product_version(self):
+        """
+        Test get_product_version testing the product_version get from brew list tag and product_version map
+        """
+
+        nvr1 = "logging-fluentd-container-v4.1.14-201908291507"
+        nvr2 = "ironic-container-v4.2.7-201911150432"
+
+        product_version_map = {"rhaos-4.1-rhel-8-candidate": "OSE-4.1-RHEL-8",
```

Comment:
hm ... is there any way we could obtain this mapping dynamically ? some fancy `brew` query cmd?

Reply:
it's just a unit test, I don't think we need something fancy here.